web-app/js/ng-app/app.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-var app = angular.module('ang', []);
\ No newline at end of file
+angular.module('ang', ['ngRoute'])
+    .config(function ($routeProvider) {
+        $routeProvider
+            .when('/book/create', {
+                templateUrl : '/templates/book/createBookView.html',
+                controller  : 'createBookController'
+            })
+            .when('/book/list', {
+                templateUrl : '/templates/book/bookListView.html',
+                controller  : 'bookController as bC'
```

Comment:
use controllerAs property instead.

rename "bC" to bookCtrl

rename all controller "classes" to PascalCase.

Reply:
Also. Move filte to **configs** folder and rename file to "routing.js"